packages/devtools_app/lib/src/screens/performance/panes/queued_microtasks/queued_microtasks_view.dart

```diff
@@ -0,0 +1,278 @@
+// Copyright 2025 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
+
+import 'package:devtools_app_shared/ui.dart';
+import 'package:devtools_app_shared/utils.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart' show DateFormat;
+import 'package:vm_service/vm_service.dart';
+
+import '../../../../shared/analytics/constants.dart' as gac;
+import '../../../../shared/primitives/utils.dart' show SortDirection;
+import '../../../../shared/table/table.dart' show FlatTable;
+import '../../../../shared/table/table_data.dart';
+import '../../../../shared/ui/common_widgets.dart' show RefreshButton;
+import 'queued_microtasks_controller.dart';
+
+class RefreshQueuedMicrotasksButton extends StatelessWidget {
+  const RefreshQueuedMicrotasksButton({
+    super.key,
+    required QueuedMicrotasksController controller,
+  }) : _controller = controller;
+
+  final QueuedMicrotasksController _controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<QueuedMicrotasksControllerStatus>(
+      valueListenable: _controller.status,
+      builder: (_, status, _) {
+        return RefreshButton(
+          iconOnly: true,
+          outlined: false,
+          onPressed: status == QueuedMicrotasksControllerStatus.refreshing
+              ? null
+              : _controller.refresh,
+          tooltip:
+              "Take a new snapshot of the selected isolate's microtask queue.",
+          gaScreen: gac.performance,
+          gaSelection: gac.PerformanceEvents.refreshQueuedMicrotasks.name,
+        );
+      },
+    );
+  }
+}
+
+class QueuedMicrotasksTabControls extends StatelessWidget {
```

Comment:
this helper widget seems unnecessary. Can we just use RefreshQueuedMicrotasksButton directly below where this widget is currently used?